tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,95 @@
+            russian_holidays = Turkmenistan(years=range(1992, 2050), language="ru")
+            for holiday in russian_holidays:
+                self.assertEqual(holiday.lang, "ru", f"Holiday {holiday.name} not localized in Russian.")
+        except FileNotFoundError:
+            print("Russian language translation files not found")
+        
+        try:
+            english_holidays = Turkmenistan(years=range(1992, 2050), language="en_US")
+            for holiday in english_holidays:
+                self.assertEqual(holiday.lang, "en_US", f"Holiday {holiday.name} not localized in English.")
+        except FileNotFoundError:
+            print("English language translation files not found")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)